tests/voq/test_voq_nbr.py

```diff
@@ -128,16 +129,19 @@ def restore_bgp(duthosts, nbrhosts, all_cfg_facts):
 
                     for peer in nbr['conf']['bgp']['peers']:
                         for neighbor in nbr['conf']['bgp']['peers'][peer]:
-                            nbr['host'].eos_config(
-                                lines=["no neighbor %s shutdown" % neighbor],
-                                parents=['router bgp {}'.format(nbr['conf']['bgp']['asn'])])
-
-                            if ":" in address:
+                            if isinstance(nbr['host'], EosHost):
                                 nbr['host'].eos_config(
-                                    lines=["no ipv6 route ::/0 %s " % neighbor])
+                                    lines=["no neighbor %s shutdown" % neighbor],
+                                    parents=['router bgp {}'.format(nbr['conf']['bgp']['asn'])])
+
+                                if ":" in address:
+                                    nbr['host'].eos_config(
+                                        lines=["no ipv6 route ::/0 %s " % neighbor])
+                                else:
+                                    nbr['host'].eos_config(
+                                        lines=["no ip route 0.0.0.0/0 %s " % neighbor])
                             else:
-                                nbr['host'].eos_config(
-                                    lines=["no ip route 0.0.0.0/0 %s " % neighbor])
+                                nbr['host'].shell("sudo vtysh -c 'configure terminal' -c 'router bgp " + str(nbr['conf']['bgp']['asn']) + "' -c 'no neighbor {} shutdown'".format(neighbor))
```

Comment:
we don't need to add default route if its sonic devices?

Reply:
Removed it from eos.